### ISSUE TITLE
Unable to turn off versioning

### ISSUE THREAD

**Issue by kilbd** [BetaHuhn/do-spaces-action#141]
Thanks for your work on this Action! It's really fast.

**🐞 Describe the bug**

I'm not interested in versioning. Setting the `versioning` parameter to `false` still creates a __latest__ folder in my Space. Leaving off the parameter results in both a __latest__ and a __v0.1.0__ version (from my package.json).

**📚 To Reproduce**

Either leave off `versioning` or set `versioning: false` in the `with` configuration of the Action.

**💡 Expected behavior**

I expected to have no versions, just the contents of `source` uploaded to my Space.

**⚙️ Environment**
 - OS: GitHub Runner/Actions, `runs-on: ubuntu-latest`
 - Action version: `@master`

**Comment by BetaHuhn** [comment]
Hey @kilbd, thanks for opening this issue!

This is definitely a bug. The action didn't parse the versioning option correctly ([`d7d37f8`](https://github.com/BetaHuhn/do-spaces-action/commit/d7d37f8c3469b1e4bb81b99417cf1357ade4d42a)), which led to the option always being turned on.

I implemented a fix on the [develop](https://github.com/BetaHuhn/do-spaces-action/tree/develop) branch.

You can try it out by using the action like this:

```yml
uses: BetaHuhn/do-spaces-action@develop
```

Let me know if this resolves your issue and I will merge this into master and release a new version ASAP.

**Comment by kilbd** [comment]
Setting `versioning: false` did what I expected and did not upload a version folder. Not sure of your intended behavior, but leaving off the `versioning` input resulted in the previous behavior of creating **latest** and **v0.1.0** versions. (Effectively, default is `versioning: true`)

**Comment by BetaHuhn** [comment]
Yes, this was the intended behavior. I can see why that's not expected though and will change it to be false by default.

**Comment by kilbd** [comment]
Thanks again for a great action!

**Comment by BetaHuhn** [comment]
:tada: This issue is resolved in version 2.0.0 :tada:

The release is available on [GitHub release](https://github.com/BetaHuhn/do-spaces-action/releases/tag/v2.0.0). It also includes some other changes and refactoring. I also updated the [README](https://github.com/BetaHuhn/do-spaces-action#readme) to make it easier to get started.

To use the latest version, use the Action with the `v2` tag:

```yml
uses: BetaHuhn/do-spaces-action@v2
```

Or the `latest` tag:

```yml
uses: BetaHuhn/do-spaces-action@latest
```
